doc/manual/manual.tex

```diff
@@ -320,11 +320,12 @@ \subsection{Referencing \aspect{}}
 }
 \end{lstlisting}
 
+and cIte the  most recent release of  the software package. See \url{https://aspect.dealii.org/publications}.
```

Comment:
cIte -> cite

But this is also an odd sentence that starts with "and".

The correct URL requires `.html` at the end.